<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_help</class>
 <widget class="QDialog" name="Dialog_help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>103</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>480</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>User Interface Components</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>431</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>1) Cryptosystem: Generate Training data using Affine ciphers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>431</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>2) Cryptanalysis: Perform frequency analysis &amp; GA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>331</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>3) Statistics:  View n-gram and fitness plots</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>361</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>5) Help: Project assistance &amp; overview </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>391</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>4) Credits: Project team &amp; supervisor details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_help</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
